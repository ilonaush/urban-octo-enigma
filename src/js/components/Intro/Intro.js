import "./Intro.styl";
import React from 'react';

const Intro = ({children, ...props}) => {
    return (
    <div className='intro'>
        {children}
    </div>
    )
};

export default Intro;
