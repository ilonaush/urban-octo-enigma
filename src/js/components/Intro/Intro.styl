@import "~js/vars";

.intro
  background linear-gradient(90deg,$dark-blue 0,$blue 60%,$light-blue)
  min-height 40px
  padding 30px
  text-transform uppercase
  color #fff

