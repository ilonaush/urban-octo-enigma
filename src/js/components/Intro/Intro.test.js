import React from 'react';
import Intro from './Intro';
import renderer from 'react-test-renderer';

describe('Intro', () => {
    it('render correctly intro component', () => {
        const IntroComponent = renderer.create(<Intro />).toJSON();
        expect(IntroComponent).toMatchSnapshot();
    });
});


