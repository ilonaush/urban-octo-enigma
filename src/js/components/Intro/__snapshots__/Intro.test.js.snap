// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Intro render correctly intro component 1`] = `
<div
  className="intro"
/>
`;
